web/packages/teleterm/src/ui/AccessRequestCheckout/useAccessRequestCheckout.ts

```diff
@@ -28,12 +28,19 @@ import {
 import { useSpecifiableFields } from 'shared/components/AccessRequests/NewRequest/useSpecifiableFields';
 
 import { CreateRequest } from 'shared/components/AccessRequests/Shared/types';
+import {
```

Comment:
Can we reexport ` isKubeClusterWithNamespaces` and `KubeNamespaceRequest` from `'shared/components/AccessRequests/NewRequest`?